DOCUMENTATION.md

```diff
@@ -2199,7 +2200,7 @@ Correct usage of a semaphore is established by programming convention in the app
 
 As an alternative, potentially safer approach to the multiple-permit acquire, you can use the `tryAcquire()` method of Semaphore.
 It tries to acquire the permits in optimistic manner and immediately returns with a `boolean` operation result. It also accepts
-an optional `timeout` argument which specifies the timeout in milliseconds to acquire the permits before giving up.
+an optional `timeout` argument that specifies the timeout in milliseconds to acquire the permits before giving up.
```

Comment:
```suggestion
an optional `timeout` argument which specifies the timeout in milliseconds to acquire the permits before giving up.
```

I don't think we need this change.

Reply:
I remembered the grammar wrong. Right, which seems better

Reply:
as I know it does not matter. that's why i said it is not needed.